themes/digital.gov/layouts/services/service-contact.html

```diff
@@ -18,21 +18,21 @@
       {{/* get the path for the icon — e.g. 18f-icon.png */}}
```

Comment:
We should refactor this page in the future. 

- Templates was the **old** way of setting partials [hugo docs](https://gohugo.io/templates/introduction/#template).
- There's a lot of repetition on how the template is being loaded, see template calls with identical parameters on lines 21, 28, and 35.

Reply:
Created issue https://github.com/GSA/digitalgov.gov/issues/5645.